### PR TITLE
rm: linux: Describe the LmP Distros

### PR DESCRIPTION
**Pull request by angolini**
Signed-off-by: Daiane Angolini <daiane.angolini@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for a7434b648d49477252f33d69d038545769c47600 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/991/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 3d48e970c1589a514face258ba0cc4fc46f7a291 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/992/docs/artifacts/html/index.html

File: source/reference-manual/linux/linux-distro.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+======
+
+LmP provides reference distros to be used in different use cases. They state the default configuration for the tools used on a FoundriesFactory. The distro definition files are under ``meta-lmp/meta-lmp-base/conf/distro``.
+
+If you are not familiar with the concept of Distro, the Yocto Project definition can help (`term-DISTRO`_), another good document is the `Poky`_ description, which is the default distro reference used by the Yocto Project.
+
+LmP
+***
+
+The ``lmp`` is the default distro for a FoundriesFactory. The main point of this distro is to configure the packages for working with OTA, and to organize qhe boot sequence, including the image architecture using OSTree and installing the needed artifacts.
```
Comment (vanmaegima) [inline source/reference-manual/linux/linux-distro.rst:13]: the* boot sequence

File: source/reference-manual/linux/linux-distro.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+LmP Base
+********
+
+The ``lmp-base`` is the distro recommended for bring up and low level development (when the platform system is being developed or adjusted, for example).
+
+It overrides some configurations from ``lmp`` to generate a system more friendly to de development, such as:
```
Comment (vanmaegima) [inline source/reference-manual/linux/linux-distro.rst:20]: to generate a friendly system for development

File: source/reference-manual/linux/linux-distro.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
+.. _ref-linux-distro:
+
+Distro
```
Comment (vanmaegima) [inline source/reference-manual/linux/linux-distro.rst:3]: maybe call it 'lmp distros'?

**Review by vanmaegima — APPROVED** [review]
minor comments but in general LGTM

**Comment by doanac** [comment]
Docs for e2f5ee8d593e7b539711d878fc0e5c3a0dd14a34 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/993/docs/artifacts/html/index.html

File: source/reference-manual/linux/linux-distro.rst [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+The ``lmp`` is the default distro for a FoundriesFactory. The main point of this distro is to configure the packages for working with OTA, and to organize the boot sequence, including the image architecture using OSTree and installing the needed artifacts.
+
+LmP Base
+********
+
+The ``lmp-base`` is the distro recommended for bring up and low level development (when the platform system is being developed or adjusted, for example).
+
+It overrides some configurations from ``lmp`` to generate a friendly system for development, such as:
+
+* It does not support OSTree.
```
Comment (ricardosalveti) [inline source/reference-manual/linux/linux-distro.rst:22]: Also good to say the rootfs is rw and can be customized by the user (and no OTA support as consequence).

**Review by ricardosalveti — APPROVED** [review]
Minor comment but LGTM

**Comment by doanac** [comment]
Docs for 5642dad13591414816ab4bfc3167479ce0146c90 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/998/docs/artifacts/html/index.html

File: source/reference-manual/linux/linux-distro.rst (hunk 4 above)
Comment (angolini) [reply source/reference-manual/linux/linux-distro.rst:22]: added. thanks!

**Pull request merged**